handout/features/es6_modules.md

```diff
@@ -1,8 +1,33 @@
-# ES6 Modules
+# ECMAScript 2015 Modules
```

Comment:
I agree that we should mention the difference between ES6, ES2015 and ECMAScript 2015 at the beginning. I wouldn't use ECMAScript 2015 everywhere because is just too long but we could use the shorthand ES2015 which people are going to get used to when working with tools like Webpack or Typescript.